Script/item.dat

```diff
@@ -1104,8 +1104,8 @@ potion /* materialcontainer-> */
   FormModifier = 25;
   NameSingular = "bottle";
   MainMaterialConfig == GLASS;
-  SecondaryMaterialConfig = { 11, WATER, HEALING_LIQUID, OMMEL_URINE, POISON_LIQUID, VALDEMAR, ANTIDOTE_LIQUID, VODKA, TROLL_BLOOD, OMMEL_SWEAT, OMMEL_TEARS, SULPHURIC_ACID; }
-  MaterialConfigChances = { 11, 75, 50, 5, 50, 1, 10, 5, 25, 5, 5, 25; }
+  SecondaryMaterialConfig = { 13, WATER, HEALING_LIQUID, OMMEL_URINE, POISON_LIQUID, VALDEMAR, ANTIDOTE_LIQUID, VODKA, TROLL_BLOOD, OMMEL_SWEAT, OMMEL_TEARS, SULPHURIC_ACID, MUSTARD_GAS_LIQUID, LIQUID_DARKNESS; }
```

Comment:
I can decrease their spawning chances, but why remove them altogether? They can be interesting when found, and nearly n players will have that reaction - I know very well that before I started to delve into script files, I didn't think about how the items I find are generated, or that I could have gotten a bottle of ommel spit instead of this poison. I don't think anyone but code-readers will be able to tell any difference.